OJS.Workers.ExecutionStrategies/ExecutionStrategy.cs

```diff
@@ -57,31 +75,41 @@ public ExecutionResult SafeExecute(ExecutionContext executionContext)
             }
         }
 
-        protected ExecutionResult CompileExecuteAndCheck(
-            ExecutionContext executionContext,
+        protected virtual IExecutionResult<OutputResult> ExecuteAgainstSimpleInput(
+            IExecutionContext<string> executionContext) =>
+                throw new NotImplementedException();
```

Comment:
Is this on purpose?
If yes, better create a custom exception to be transparent.